__tests__/secret.test.js

```diff
@@ -89,4 +89,15 @@ describe('Tozny identity client', () => {
     expect(ops.createSecret(secretNameInvalid)).rejects.toThrow()
     expect(ops.createSecret(secretValueEmpty)).rejects.toThrow()
   })
+  it('can create a secret, and list it', async () => {
+    const secret = {
+      secretType: 'Credential',
+      secretName: `test-secret-${uuidv4()}`,
+      secretValue: 'secret-value',
+      description: 'this is a description',
+    }
+    await ops.createSecret(realmConfig, identity, secret)
+    await new Promise(r => setTimeout(r, 1000))
+    await ops.getSecrets(realmConfig, identity, 10)
```

Comment:
Do we validate our expected secrets come back in the search?

Reply:
It returns a query so its hard to write a "expects" but yes it does come back